hrqb/cli.py

```diff
@@ -44,6 +46,41 @@ def ping(ctx: click.Context) -> None:
     )
 
 
+@main.command()
+@click.pass_context
+def test_connections(ctx: click.Context) -> None:
+    """Test connectivity with Data Warehouse and Quickbase."""
+    dw_success = False
+    qb_success = False
+
+    try:
+        if DWClient().test_connection():
+            logger.debug("Data Warehouse connection successful")
+            dw_success = True
+    except Exception as exc:  # noqa: BLE001
+        message = f"Data Warehouse connection failed: {exc}"
+        logger.error(message)  # noqa: TRY400
+    try:
+        if QBClient().test_connection():
+            logger.debug("Quickbase connection successful")
+            qb_success = True
+    except Exception as exc:  # noqa: BLE001
+        message = f"Quickbase connection failed: {exc}"
+        logger.error(message)  # noqa: TRY400
+
+    if dw_success and qb_success:
+        logger.info("All connections OK")
+    else:
+        logger.info("One or more connections failed")
```

Comment:
Optional: is this actually necessary since either the DW or QB connection message would fire before this so the logs would already have a failure message from one/both of them?

Reply:
Also considered.  They are debugging levels, this is info.  Thinking is that this, at least, will always be present.  So if we automated reading the logs, can just rely on a single string to look for.